certified-connectors/Talkdesk/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "YourTalkdeskClientId",
+          "scopes": [
+            "contacts:read agents-bulk:write callback:write notifications:write industries-activity:write users:read presence-user:read account-custom-status:read ccaas-user-status:write webhooks-trigger:write webhooks-schema:read account:read presence-account:read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://YourTalkdeskAccount.talkdeskid.com/oauth/authorize"
```

Comment:
@StefanCRT22 All the customParameter values are required until unless they have client id or secret information.

Reply:
@StefanCRT22 These values are updated to an empty string, can you please let us know the reason for this, and please note that we require all these custom parameter values.

Reply:
I have re-added the customParameters values because a new user who is going to use the Talkdesk Connector must first have a Talkdesk account. Once he has one, he will need to be able to edit the customParameters values and replace {{YourTalkdeskAccount}} with his newly created Talkdesk account name.

Reply:
@StefanCRT22 The customParameters such as `"authorizationUrl"`, `"tokenUrl"` and `"refreshUrl"` properties need exact and valid values. It is not allowed to make them dynamic. Please provide valid values to them.